zwave_js_server/model/node.py

```diff
@@ -62,6 +62,10 @@ def __init__(self, client: "Client", data: NodeDataType) -> None:
         """Initialize the node."""
         super().__init__()
         self.client = client
+        self.update_data(data)
+
+    def update_data(self, data: NodeDataType) -> None:
```

Comment:
We can't call this twice, all old references to values will be lost because we re-create it all. We should update values where possible I think ?

Reply:
I was thinking about that too just now... I'll adjust it.

Reply:
changed it, up for another review @balloob ?